tests/integration/targets/vmware_vm_portgroup_info/run.yml

```diff
@@ -0,0 +1,27 @@
+- hosts: localhost
+  gather_facts: no
+  collections:
+    - community.general
+  tasks:
+
+    - name: Import eco-vcenter credentials
+      ansible.builtin.include_vars:
+        file: ../../integration_config.yml
+      tags: eco-vcenter-ci
+
+    - name: Import simulator vars
+      ansible.builtin.include_vars:
+        file: vars.yml
+      tags: integration-ci
+
+    - name: Rest
+      ansible.builtin.import_role:
+        name: prepare_rest
+      tags: integration-ci
+
+    - name: VM list group by clusters
+      ansible.builtin.import_role:
+        name: vmware_vm_list_group_by_clusters_info
```

Comment:
this needs to be changed to `vmware_vm_portgroup_info`
also the title

Reply:
fixed thanks